src/nbl/video/device_capabilities/device_limits.json

```diff
@@ -1914,7 +1912,7 @@
         {
             "type": "uint32_t",
             "name": "lineSubPixelPrecisionBits",
-            "value": 0,
+            "value": 4,
```

Comment:
do we require line drawing feature though?

Reply:
not required
```cpp
        if (isExtensionSupported(VK_EXT_LINE_RASTERIZATION_EXTENSION_NAME))
            properties.limits.lineSubPixelPrecisionBits = lineRasterizationProperties.lineSubPixelPrecisionBits;
```

Reply:
so leave as 0